backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -318,8 +318,9 @@ object BackendSettings extends BackendSettingsApi {
             case _ =>
           }
           windowExpression.windowFunction match {
-            case _: RowNumber | _: AggregateExpression | _: Rank | _: CumeDist | _: DenseRank |
-                _: PercentRank | _: NthValue | _: NTile | _: Lag | _: Lead =>
+            case _: RowNumber | _: Rank | _: CumeDist | _: DenseRank | _: PercentRank |
+                _: NthValue | _: NTile | _: Lag | _: Lead =>
+            case aggrExpr: AggregateExpression if !aggrExpr.isInstanceOf[ApproximatePercentile] =>
```

Comment:
@rui-mo could you help velox changes, thanks !

Reply:
I think we need the match against the aggregate function like below, thanks.

> case aggExpresion: AggregateExpression if aggExpresion.aggregateFunction.isInstanceOf[ApproximatePercentile] => false

Reply:
Thanks! It is fixed now.